types/mocha/index.d.ts

```diff
@@ -170,9 +171,29 @@ declare namespace Mocha {
         currentTest: ITest;
     }
 
+    interface IStats {
+        suites: number;
+        tests: number;
+        passes: number;
+        pending: number;
+        failures: number;
+        start: Date | undefined;
+        end: Date | undefined;
+        duration: Date | undefined;
+    }
 
     /** Partial interface for Mocha's `Runner` class. */
-    interface IRunner { }
+    interface IRunner {
+        stats: IStats | undefined;
+        started: boolean;
+        suite: ISuite;
+        total: number;
+        failures: number;
+        grep: (re: string, invert: boolean) => this;
+        grepTotal: (suite: ISuite) => number;
+        globals: (arr: ReadonlyArray<string>) => this | string[];
+        abort: () => this;
+    }
```

Comment:
I think you also need [run](https://github.com/mochajs/mocha/blob/master/lib/runner.js#L809)

Reply:
You're right. I've added it 👍